install/update.php

```diff
@@ -1610,7 +1610,9 @@ function do_entities_migration($migration) {
          $DB->update(
             'glpi_plugin_fusioninventory_entities', [
                'agent_base_url' => $agent_base_url
-            ], []
+            ], [
+               '1 = 1'
```

Comment:
prefer use `[1 => 1]` or `['1' => '1']` instead `'1 = 1'`

Reply:
Ok corrected

About other where condition as there is a parameter , I presume 1 = 1 clause should be wrong